README.md

```diff
@@ -47,5 +46,5 @@ The recommended way of running is to pull the image from [Docker Hub](https://hu
 By default, the easiest way to get running is:
 
 ```bash
-docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ACCESS_URI='http://192.168.0.10:8000' -e ESPN_USER='...' -e ESPN_PASS='...'  m0ngr31/eplustv
-```
\ No newline at end of file
+docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ESPN_USER='...' -e ESPN_PASS='...'  m0ngr31/eplustv
```

Comment:
single quotes apparently don't work on Windows

```suggestion
docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ESPN_USER="..." -e ESPN_PASS="..."  m0ngr31/eplustv
```

Reply:
See https://community.getchannels.com/t/espn-with-custom-channels/31144/38?u=tmm1

Reply:
Oh, I didn't realize this had been posted there :) I love Channels!